db/active_replicator.go

```diff
@@ -114,7 +115,7 @@ func connect(idSuffix string, config *ActiveReplicatorConfig) (blipSender *blip.
 	bsc.loggingCtx = context.WithValue(context.Background(), base.LogContextKey{},
 		base.LogContext{CorrelationID: config.ID + idSuffix},
 	)
-
+	blipContext.InsecureSkipVerify = config.InsecureSkipVerify
```

Comment:
What is in `go-blip` that requires this flag? Looks like the manifest change and the branch on GitHub is missing for this to be built properly.

Typically Go libraries take in a custom `*http.Client` instead of specific flags if they need certain HTTP Client behavior overridden.

Reply:
Makes sense, as discussed added changes to pass insecureSkipVerify as an additional argument to blipSync.